src/components/InputForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import Styled from 'styled-components';
+import Button from './Button';
+
+const SearchGroup = Styled.form`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 502px;
+  height: 32px;
```

Comment:
Width and height don't need to be set here. The content should take care of that

Reply:
The buttons are rendering with some minimum differences, so I had to wrap them